src/CoreBundle/private/library/classes/TCMSTableEditor/TCMSTableWriter.class.php

```diff
@@ -127,19 +135,76 @@ protected function PostSaveHook(&$oFields, &$oPostTable)
     {
         parent::PostSaveHook($oFields, $oPostTable);
 
+        $newTableName = $oPostTable->sqlData['name'];
+
         // reset table list cache
         if (array_key_exists('_listObjCache', $_SESSION)) {
             $_SESSION['_listObjCache'] = array();
         }
 
         $this->getAutoclassesCacheWarmer()->updateTableById($this->sId);
 
-        // check table engine
-        $realEngine = $this->getRealTableEngine($oPostTable->sqlData['name']);
+        $this->adaptTableEngine($oPostTable, $newTableName);
+        $this->adaptTableName($this->sOldTblName, $newTableName);
+        $this->adaptTableComment($newTableName, $this->oldTableComment);
```

Comment:
I wouldn't pass an internal property to a private method, as the method can read it as well.